software/htmly.yml

```
@@ -0,0 +1,13 @@
+name: "HTMLy"
+website_url: "https://www.htmly.com/"
+source_code_url: "https://github.com/danpros/htmly/"
+description: "HTMLy is an open source databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
```

```suggestion
description: "Databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
```

Improved description, as per https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines